packages/utils/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+import * as os from 'os';
+import * as path from 'path';
+
+export const PIPCOOK_HOME_PATH = path.join(os.homedir(), '.pipcook');
+export const PIPCOOK_PLUGINS = path.join(PIPCOOK_HOME_PATH, 'plugins');
+export const PIPCOOK_LOGS = path.join(PIPCOOK_HOME_PATH, 'logs');
+export const PIPCOOK_DEPENDENCIES = path.join(PIPCOOK_HOME_PATH, 'dependencies');
```

Comment:
How about aggregate these paths to a named constant such as 'PIPCOOK_PATH'?

Reply:
Good idea. Fixed